lib/modules/dosomething/dosomething_kudos/includes/KudosTransformer.php

```diff
@@ -3,31 +3,28 @@
 class KudosTransformer extends Transformer {
 
   /**
-   * @param $parameters
+   * Display collection of the specified resource.
    *
+   * @param array $parameters Filter parameters to limit collection based on specific criteria.
    * @return array
    */
   public function index($parameters) {
     $filters = [
-      'reportbackitem_id' => $this->formatData($parameters['reportbackitem_ids']),
+      'fid' => $this->formatData($parameters['reportbackitem_ids']),
```

Comment:
We are, but this is what gets passed into the query, which involves the column names etc all still containing drupal ids. I've attempted to find a balance that makes sense as to where I need to switch back to Drupal ids... I could go a step farther and it only switches back to Drupal IDs at the point the query is built in https://github.com/DoSomething/dosomething/pull/4708/files#diff-2af8eaa58f7c9cf6c95554e8bb26bdfeL42